test/testdrive/top-1-monotonic.td

```diff
@@ -15,6 +15,9 @@
 $ postgres-execute connection=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
 ALTER SYSTEM SET enable_create_source_denylist_with_options = true
 
+$ postgres-execute connection=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
+ALTER SYSTEM SET min_timestamp_interval = '100ms'
```

Comment:
Can you add a test somewhere that sets the max_timestamp interval (unless I missed it)?